zombienet_tests/parachains/0002-parachains-upgrade-smoke-test.feature

```diff
@@ -0,0 +1,11 @@
+Description: Smoke Test
+Network: ./0002-parachains-upgrade-smoke-test.toml
+Creds: config
+
+
+alice: is up
+bob: is up
+alice: parachain 100 is registered within 225 seconds
+alice: parachain 100 block height is at least 10 within 400 seconds
+alice: parachain 100 perform dummy upgrade within 200 seconds
```

Comment:
What's the wasm blob that is used for the upgrade?

Reply:
Hi @drahnr, for this use case we generate a new `wasm` from the previous one by appending a `custom section`. 

A high level process definition:

```
Zombienet will connect to the collator and get the :code, after decompress a custom section is appended to the wasm and re-compressed. The new compressed code is used to perform the upgrade and then zombienet will try to match the new hash in the <node name> defined for x amount of seconds.
```

Also,  as mention in the `pr` this test is using `polkadot-collator` and `polkadot` **latest** image versions because I try with the image build within the `pr` but doesn't work with the collator version.

Thx!